Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash

```diff
@@ -0,0 +1,271 @@
+            "",
+            !__quest_state["Level 11 Desert"].state_boolean["Killing Jar Given"] &&
+                get_property_int("desertExploration") < 100 &&
+                available_amount($item[killing jar]) < 1,
+            locationAvailable($location[The Haunted Library])
+        ),
+        new Wishable(
+            $item[none],
+            $effect[Dirty Pear],
+            HTMLGenerateSpanFont("double sleaze damage", "purple"),
+            get_property_int("zeppelinProtestors") < 80,
+            true
+        ),
+        new Wishable(
+            $item[none],
+            $effect[Painted-On Bikini],
+            HTMLGenerateSpanFont("+100 sleaze damage", "purple"),
+            get_property_int("zeppelinProtestors") < 80,
+            true
+        ),
+        new Wishable(
+            $item[glark cable],
+            $effect[none],
+            "",
+            __quest_state["Level 11 Ron"].mafia_internal_step < 5,
+            locationAvailable($location[The Red Zeppelin])
+        ),
+        new Wishable(
+            $item[short writ of habeas corpus],
+            $effect[none],
+            "",
+            !__quest_state["Level 11 Hidden City"].finished,
+            locationAvailable($location[The Hidden Park])
+        ),
+        new Wishable(
+            $item[lion oil],
+            $effect[none],
+            "",
+            $item[mega gem].available_amount() < 1 &&
+                $item[lion oil].available_amount() < 1,
+            locationAvailable($location[Whitey's Grove])
+        ),
+        new Wishable(
+            $item[bird rib],
+            $effect[none],
+            "",
+            $item[mega gem].available_amount() < 1 &&
+                $item[bird rib].available_amount() < 1,
+            locationAvailable($location[Whitey's Grove])
+        ),
+        new Wishable(
+            $item[drum machine],
+            $effect[none],
+            "",
+            get_property_int("desertExploration") < 100 &&
+                $item[drum machine].available_amount() < 1,
+            locationAvailable($location[The Oasis])
+        ),
+        new Wishable(
+            $item[green smoke bomb],
+            $effect[none],
+            "",
+            __quest_state["Level 12"].state_string["Side seemingly fighting for"] != "hippy",
+            !__quest_state["Level 12"].finished &&
+                locationAvailable($location[The Battlefield (Frat Uniform)])
+        ),
+        new Wishable(
+            $item[star chart],
+            $effect[none],
+            "",
+            !__quest_state["Level 13"].state_boolean["Richard's star key used"] &&
+                $item[Richard's star key].available_amount() < 1 &&
+                $item[star chart].available_amount() < 1,
+            locationAvailable($location[The Hole In The Sky])
+        ),
+        new Wishable(
+            $item[none],
+            $effect[Frosty],
+            "init/item/meat for 8-bit",
```

Comment:
probably want to say "8-bit (among other things)" -- didn't really need it for 8-bit but it saved almost 2 turns on nuns.